test/integration/ForkTest.t.sol

```diff
@@ -227,16 +318,35 @@ contract ForkTest is Test {
         nodeDelegator1.depositAssetIntoStrategy(asset);
     }
 
-    function unpauseStrategy(address strategyAddress) private {
-        vm.startPrank(Addresses.EIGEN_UNPAUSER);
-
+    /// @dev unpause an EigenLayer Strategy is currently paused
+    function unpauseStrategy(address strategyAddress) internal {
```

Comment:
These unpause things are unrelated to the rest of the PR?

Reply:
We needed to unpause EigenLayer to test the deposits. Not a problem now as its open but it will close again.

Reply:
Makes sense!